tools/validators/instance_validator/DOAS.yaml

```diff
@@ -0,0 +1,91 @@
+CONFIG_METADATA:
```

Comment:
this file is not needed as part of the ontology update. please remove.

Reply:
The file has been deleted...